### PR TITLE
Add health check scripts for the security services

### PR DESCRIPTION
**Pull request by anonymouse64**
With https://github.com/edgexfoundry/docker-edgex-volume/pull/11, we will now have service definitions for the various security setup services. This PR adds the checks referenced from that PR to docker-edgex-consul image so that when Consul runs it is able to run these scripts as needed.

The next step after this towards fixing #1710 is to overwrite the entrypoint/command for the security services in docker-compose to wait on their dependencies with the pattern we have already established for Kong and related services here. The PR doing that is open here: https://github.com/edgexfoundry/developer-scripts/pull/172. However, that PR also has a dependency on edgex-go to change the security-proxy-setup-go Docker image to use Alpine so we can run `/bin/sh` and `curl`, as well as a change adding `curl` to the security-secretstore-setup Docker image. That PR is https://github.com/edgexfoundry/edgex-go/pull/2075.

The developer-scripts PR at https://github.com/edgexfoundry/developer-scripts/pull/172 will have full testing instructions for how to test this change along with the other PR's.

See https://github.com/edgexfoundry/edgex-go/issues/1710 for full details on this task.

### CODE REVIEW

**Comment by anonymouse64** [comment]
CC @tingyuz @jim-wang-intel @bnevis-i

File: scripts/security-proxy-setup-checker.sh
Comment (anonymouse64) [inline scripts/security-proxy-setup-checker.sh]: delete this as per @bnevis-i

**Review by bnevis-i — DISMISSED** [review]
Reviewed at F2F and approve with or without suggested tweak to remove redundant `/consul/scripts/consul-svc-healthy.sh security-secrets-setup` check.

File: scripts/security-proxy-setup-checker.sh
Comment (anonymouse64) [reply scripts/security-proxy-setup-checker.sh]: @bnevis-i done

**Comment by anonymouse64** [comment]
> Reviewed at F2F and approve with or without suggested tweak to remove redundant /consul/scripts/consul-svc-healthy.sh security-secrets-setup check.

I removed it, but we need someone else with write access here to approve before I can merge

**Comment by bnevis-i** [comment]
cc: @ernestojeda @soda480

**Review by tingyuz — APPROVED** [review]
LGTM.

**Pull request merged**
